Plugins/Feedback/NWScript/nwnx_feedback.nss

```diff
@@ -52,6 +52,31 @@ int NWNX_Feedback_GetCombatLogMessageHidden(int nMessage, object oPC = OBJECT_IN
 // to TRUE but the personal state is set to FALSE, the message will be shown to oPC
 void NWNX_Feedback_SetCombatLogMessageHidden(int nMessage, int nState, object oPC = OBJECT_INVALID);
 
+// Gets if journal updated message nMessage is hidden.
+// Notes:
+// If oPC == OBJECT_INVALID it will return the global state:
+//    TRUE      nMessage is globally hidden
+//    FALSE     nMessage is not globally hidden
+// If oPC is a valid player it will return the personal state:
+//    TRUE      nMessage is hidden for oPC
+//    FALSE     nMessage is not hidden for oPC
+//    -1        Personal state is not set
+int NWNX_Feedback_GetJournalUpdatedMessageHidden(object oPC = OBJECT_INVALID);
```

Comment:
Should probably change all the `nMessage is` references to `Journal updated messages are` or something